README.md

```diff
@@ -1 +1,54 @@
-# designing_microservices_course
\ No newline at end of file
+# Designing microservices course
+This repository is intended for demonstration purposes for students during the "Basics of microservices architecture 
+course" which is going to be taught in 2019 autumn term for 4 years students if ITMO university, department of 
```

Comment:
pls find exact name of your department/faculty in ITMO webpage since I haven't found "department of informational systems". Or better just skip this information :)